src/qibo/models/circuit.py

```diff
@@ -72,7 +72,7 @@ def QFT(nqubits: int, with_swaps: bool = True,
     return circuit
 
 
-def _DistributedQFT(nqubits, accelerators=None):
+def _DistributedQFT(nqubits, accelerators=None): # pragma: no cover
```

Comment:
Thanks for the updates. In principle this method and the one in `test_circuit_constructor` should be tested virtually on CPU with qibotf. If it is not, there may be something strange going on. I am checking in another branch if coverage will pass without these exceptions.